aldryn_client/cli.py

```diff
@@ -272,6 +273,26 @@ def boilerplate_upload(ctx):
     click.echo(ret)
 
 
+@cli.command()
+def version():
+    """Show version info"""
+    from . import __version__
+    click.echo('package version: {}'.format(__version__))
+
+    try:
+        # try to get git revision
+        script_home = os.path.dirname(__file__)
+        git_dir = os.path.join(script_home, '..', '.git')
+        revision = subprocess.check_output([
+            'git', '--git-dir', git_dir,
+            'rev-parse', '--short', 'HEAD'
+        ]).strip()
+        click.echo('git revision:    {}'.format(revision))
+    except:
```

Comment:
If you're going to catch all exceptions and suppress them, maybe at least add a comment explaining why this is a good thing to do? It usually isn't.